packages/core/components/Alert/Alert.tsx

```diff
@@ -45,21 +47,32 @@ export type AlertProps = Omit<
 > & DialogImageProps & {
   message: string;
   closeAlert: ModalProps['closeModal'];
+  hasSound: boolean;
```

Comment:
```suggestion
  hasSound?: boolean;
```

It would be better if this prop was optional. Don't you think?